pkg/cmd/roachtest/github.go

```diff
@@ -112,7 +134,7 @@ func (g *githubIssues) createPostRequest(t test.Test, message string) issues.Pos
 		MentionOnCreate: mention,
 		ProjectColumnID: projColID,
 		PackageName:     "roachtest",
-		TestName:        t.Name(),
+		TestName:        issueName,
```

Comment:
I wish we didn't have to do this. Ideally, I'd want the test failure to be: `roachtest: TestFoo failed (ssh_problem)` instead of the cryptic `roachtest: ssh_problem failed`. We will have to go through the artifacts each time to find out which test actually failed.

I'd suggest we don't overwrite the test name, except that for the `cluster_creation` error, if the cloud provider is having issues, typically lots of test will fail and we don't want to create a separate issue for each one.

Reply:
It probably makes sense to keep SSH flakes together, instead of having a new one created for each incidence. 

We can prepend the test name to the message body like cluster creation does?